templates/common/infra/bicep/core/search/search-services.bicep

```diff
@@ -0,0 +1,41 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+param sku object = {
+  name: 'standard'
+}
+
+param authOptions object = {}
+param semanticSearch string = 'disabled'
+
+resource search 'Microsoft.Search/searchServices@2021-04-01-preview' = {
```

Comment:
This is for future consideration, but I see it useful because we've added a lot of opinionated defaults.

For new resources like these that have an opinionated default, do you think we can describe in the bicep module metadata, what this type of resource this is meant to create? For example: `private-searchServices` which means it's a more secure thing.

Reply:
Yes, something to consider for future modules.